zkevm-circuits/src/tx_circuit/test.rs

```diff
@@ -114,10 +114,9 @@ fn run<F: Field>(
     max_txs: usize,
     max_calldata: usize,
 ) -> Result<(), Vec<VerifyFailure>> {
-    let k = max(
-        20,
-        log2_ceil(TxCircuit::<F>::min_num_rows(max_txs, max_calldata)),
-    );
+    let active_row_num = TxCircuit::<F>::min_num_rows(max_txs, max_calldata);
+
+    let k = max(20, log2_ceil(active_row_num));
```

Comment:
k = `min(20, log2_ceil(active_row_num)` ?

Reply:
sig circuit (embedded inside tx circuit) use LOG_NUM_ROWS=20, so at least need 20?

Reply:
yeah, you're right.